SwrveSDK/src/main/java/com/swrve/sdk/conversations/SwrveConversation.java

```diff
@@ -24,6 +24,8 @@
     protected transient SwrveBase<?, ?> swrve;
     // Parent in-app campaign
     protected transient SwrveConversationCampaign campaign;
+    // Priority of the conversation
+    protected int priority = 9999;
```

Comment:
I'd put this in the SwrveBaseConversation, alongside the id/name/pages variables. Also move the getter there, but do not provide a setter for this.

Why the default value of 9999?